plugin/batch-queue/job_examples/stampede/slurm_launch.job

```diff
@@ -0,0 +1,103 @@
+#!/bin/bash
+# Put your SLURM options here
+#SBATCH --time=00:30:00          # put proper time of reservation here
+#SBATCH --nodes=129              # number of nodes, where one node is the coordinator node, and the rest are the computing nodes
+#SBATCH --ntasks-per-node=16     # processes per node
+#SBATCH --job-name="example"
+#SBATCH --output=example.std
+#SBATCH --error=example.err
+#SBATCH --partition=normal
+#SBATCH -A project_number
+# ? Any other batch options ?
+
+# Start dmtcp coordinator on launching node. Free TCP port is automatically allocated.
+# this function creates dmtcp_command.$JOBID script that serves like a wrapper around
+# dmtcp_command that tunes it on exact dmtcp_coordinator (it's hostname and port)
+# instead of typing "dmtcp_command -h <coordinator hostname> -p <coordinator port> <command>"
+# you just type "dmtcp_command.$JOBID <command>" and talk to coordinator of JOBID job
```

Comment:
you just type -> one just types
and talk -> and talks
to coordinator of -> to the coordinator
job -> job.